### PR TITLE
Update common_solver_opts.md

### PR DESCRIPTION
**Pull request by Datseris**
Just small tidying up and making this just a bit clearer

### CODE REVIEW

File: docs/src/basics/common_solver_opts.md [hunk 1]
```diff
@@ -63,21 +71,15 @@ For more examples for controlling the output behavior, see the
...
-  (to save memory) if one isn't using interpolations when a custom callback is
-  used. Another case where this may be used is to turn on interpolations for
-  usage in the integrator interface even when interpolations are used no
-   where else. Note that this is only required if the algorithm doesn't have
-  a free or lazy interpolation (`DP8()`).
 * `save_everystep`: Saves the result at every timeseries_steps iteration.    
   Default is true if `isempty(saveat)`.
 * `timeseries_steps`: Denotes how many steps between saving a value for the
-  timeseries. Defaults to 1.
+  timeseries. These "steps" are the steps that the solver stops internally, not the ones that are
```
Comment (ChrisRackauckas) [inline docs/src/basics/common_solver_opts.md]: Maybe easier to somehow mention that this is for `save_everystep`

**Comment by Datseris** [comment]
Yes indeed. a simple parenthesis did the trick

**Comment by ChrisRackauckas** [comment]
I think that this page

https://github.com/JuliaDiffEq/DiffEqDocs.jl/blob/master/docs/src/features/output_specification.md

Should either be updated simultaneously, or deemed unnecessary. It should just be a page with a bunch of examples for how to do various saving combinations, but maybe that's not needed if the common solver options page is cleared up.

**Comment by Datseris** [comment]
I wanted to talk to you about that. I think it is redundant. It's better to have another section (with one hashtag) on the page common solver options that will be titled examples and would have 2 or 3 examples of specified output... I will do this on this PR.

**Comment by ChrisRackauckas** [comment]
Sounds like a good idea.

**Comment by Datseris** [comment]
Okay done that as well. I think that's all.

File: docs/src/basics/common_solver_opts.md [hunk 2]
```diff
@@ -177,13 +177,15 @@ options control the errors which are calculated:
 # Examples 
 The following lines are examples of how one could use the configuration of `solve()`. For these examples a 3-dimensional ODE problem is assumed, however the extention to other types is straightforward. 
 
-1. `solve(prob, AlgorithmName())` : The "default" setting. All parameters get their default values. 
-  This means that the solution is saved at the steps the Algorithm stops internally and dense output is enabled. 
+1. `solve(prob, AlgorithmName())` : The "default" setting, with a user-specified algorithm (given by `AlgorithName()`).
```
Comment (ChrisRackauckas) [inline docs/src/basics/common_solver_opts.md]: AlgorithName typo

**Comment by Datseris** [comment]
fixed

**Pull request merged**